### PR TITLE
docs: fix link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by maxhbr**
## Summary

this just fixes a link in CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-See the [Contribute section](https://docs.magmacore.org/docs/contributing/contribute_onboarding)
-of Magma documentation for information on developer onboarding, workflow, 
+See the [Contribute section](https://github.com/magma/magma/wiki/Contributing-Code/)
```
Comment (ildikov) [inline CONTRIBUTING.md]:
```suggestion
See the [Contributor Guide](https://github.com/magma/magma/wiki/Contributing-Code/)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+of Magma documentation for information on developer onboarding, workflow,
```
Comment (ildikov) [inline CONTRIBUTING.md]:
```suggestion
for information on developer onboarding, workflow,
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
 contribution convention, and presubmit checks.
```
Comment (ildikov) [inline CONTRIBUTING.md]:
```suggestion
contribution conventions, presubmit checks and more.
```

**Review by themarwhal — APPROVED** [review]

**Review by ildikov — CHANGES_REQUESTED** [review]
Just a few minor suggestions in line.

Thanks,
Ildikó

**Review by ildikov — APPROVED** [review]
Thanks for the quick fix!

Best Regards,
Ildikó

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
-See the [Contribute section](https://docs.magmacore.org/docs/contributing/contribute_onboarding)
-of Magma documentation for information on developer onboarding, workflow, 
-contribution convention, and presubmit checks.
+See the [Contributor Guide](https://github.com/magma/magma/wiki/Contributing-Code/)
```
Comment (ildikov) [inline CONTRIBUTING.md]:
```suggestion
See the [Contributor Guide](https://github.com/magma/magma/wiki/Contributor-Guide)
```

**Review by ildikov — CHANGES_REQUESTED** [review]
Just one more thing. I think we should point to the root of the Contributor Guide so anyone who finds the link here can navigate themselves what information they need rom there.

**Review by ildikov — APPROVED** [review]
Looks great, thank you!

**Pull request merged**